modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/ListTypeDefinitionResourceTest.java

```diff
@@ -125,12 +118,49 @@ public void testPostSpecificationListTypeDefinition() throws Exception {
 		return _cpSpecificationOption.getCPSpecificationOptionId();
 	}
 
+	protected ListTypeDefinition
+			testGetSpecificationIdListTypeDefinitionsPage_addListTypeDefinition(
+				Long id, ListTypeDefinition listTypeDefinition)
+		throws Exception {
+
+		return _addListTypeDefinition(randomListTypeDefinition());
```

Comment:
is it correct that here we generate another random and not use the parameter one?